plugin/src/main/kotlin/com/jetbrains/packagesearch/plugin/ui/bridge/Components.kt

```diff
@@ -130,3 +139,35 @@ internal fun PackageActionPopup(
         }
     }
 }
+
+
+
+internal fun SplitPaneScope.defaultPKGSSplitter(
```

Comment:
Do not use PKGS, also the word default is pointless here. `packageSearchSplitter` is more suitable.